File: AD-HOC/3315.js
let maior = 0;

for (let i = 0; i < 4; i++) {
  let linha = lines.shift().split(' ').map(Number);
  let somador = 0;
  
  for (let j = 0; j < linha.length; j++) {
    somador += linha[j];
  }

  if (somador > maior) {
    maior = somador;
  }
}

function binario(n){
  return (n >>> 0).toString(2);
}

console.log(`${maior} = ${binario(maior)}`);

File: AD-HOC/3315.py
maior = 0

for i in  range(4):
  linha = list(map(int, input().split()))
  somador = 0
  
  for j in range(len(linha)):
    somador += linha[j]

  if (somador > maior):
    maior = somador

binario = str(bin(maior))

print("{} = {}".format(maior, binario[2:]))
